src/azure-cli/azure/cli/command_modules/resource/_bicep.py

```diff
@@ -177,6 +196,38 @@ def supports_bicep_publish():
     return semver.compare(installed_version, "0.4.1008") >= 0
 
 
+def _bicep_installed_in_ci():
+    if "GITHUB_ACTIONS" in os.environ or "TF_BUILD" in os.environ:
+        from shutil import which
+
+        return which("bicep") is not None
```

Comment:
Worth adding logging here to distinguish between "env vars not set" & "bicep not found on the PATH"?

Reply:
Makes sense.